capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.upcase
```

Comment:
- [ ] You have done an excellent job, congrats on it! 🎉 However, the goal of this decorator is to capitalize the string, but when you use `upcase` you are actually converting everything into uppercase e.g:

capitalize -> My string
upcase -> MY STRING